src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/multiline/options/GTTestsOptionPanelMSAMultiline.cpp

```diff
@@ -0,0 +1,2571 @@
+/**
```

Comment:
Please add new and unique tests for multi-line mode. 

Do not copy existing ones with 1 extra line added to open multi-line view. Most of them are not related to the multi-line mode at all.

Please think about quality, but not about quantity.

Reply:
Can't agree now
The single mode tests must be passed in the multi-line mode too
I have found ~30 bugs by this simple feature
Let's run for a while this duplicated tests
I will create additional explicit multi-line tests by the fail
After merge to a master we will switch the feature off

Reply:
If you want to keep some tests which are 100% copy-paste (except enableMultiline)  you can 

- Extract the test method into a function and call it from all duplicated places. 
or
- Wrap the original test code with 'for mode in [multiline, non-multilne]' loop - it still be a single test but it will test both modes.

This will be very explicit and we will have no code duplication in this case.